tests/integration/dom0-template/etc/snmp/snmp.xs.conf

```diff
@@ -0,0 +1,49 @@
+[common]
+enabled=yes
+debug_log=yes
+
+[snmpd]
+v2c=yes
+v3=yes
+
+[snmpd_v2c]
+community=snmpv2cpublic
+
+[snmpd_v3]
+security_name=test123
+authentication_passphrase=********
+authentication_protocol=SHA
+privacy_passphrase=********
+privacy_protocol=AES
+
+engine_id=0x80001f88800fceda7f01824c6500000000
+
+[snmpd_v3_internal]
+name_key="test123"
+security_name_key="test123"
+authentication_key=0xsnmpv3authenticationkey
+authentication_protocol_oid=.1.3.6.1.6.3.10.1.1.3
+privacy_key=0xsnmpv3privacykey
+privacy_protocol_oid=.1.3.6.1.6.3.10.1.2.4
+
+[nms]
+nms_address = 192.168.1.1
+nms_port = 162
+ 
+[snmptrap]
+v2c = yes
+v3 = yes
+ 
+[snmptrap_v2c]
+community = snmpv2cpublic
+ 
+[snmptrap_v3]
+user_name = xs_snmp_user
+authentication_password = ********
+authentication_protocol = MD5
+privacy_password = ********
+privacy_protocol = DES
+ 
```

Comment:
```suggestion

```
Remove space from empty line. I'll submit a fixup using pre-commit to remove trailing whitespace for future PRs:
https://pre-commit.com/#2-add-a-pre-commit-configuration